src/layoutPicker.ts

```diff
@@ -18,10 +18,8 @@ class LayoutItem implements QuickPickItem {
  */
 const layouts: QuickPickItem[] = [
   new LayoutItem("dagre", "The directed graph layout library Dagre"),
-  new LayoutItem(
-    "elk",
-    "Eclipse Layout Kernel (ELK) with the Layered algorithm"
-  ),
+  new LayoutItem("elk", "Eclipse Layout Kernel (ELK) with the Layered algorithm"),
+  new LayoutItem("tala", "General Orthogonal Layout Engine")
```

Comment:
is there a way to only add this if it's found in the user's path? most users won't have it installed and it'll give an error when they try to use it

Reply:
Since I don't have TALA installed, what would I be looking for (windows/macos/linux).  I can easily change the quick pick, and I might be able to change the package settings on the fly (they try to lock it down, but...)

Reply:
ohh interesting, it's something written in compile time?

in that case, can we check that they have it installed upon selecting, and if they don't, revert the selection and show an error msg?

The binary is `d2plugin-tala`

Reply:
Is it d2plugin-tala.dll on Windows?

Reply:
no, exe i think